docs/docs/data-sources/dynamodb.md

```diff
@@ -0,0 +1,37 @@
+---
+sidebar_position: 11
+---
+
+# DynamoDB
+
+ToolJet can connect to DynamoDB to read and write data.
+
+## Connection
+
+Please make sure the host/ip of the database is accessible from your VPC if you have self-hosted ToolJet. If you are using ToolJet cloud, please whitelist our IP.
```

Comment:
This might also be not relevant in the case of DynamoDB. Let's remove it for now.

Reply:
Done